README

```diff
@@ -935,8 +937,32 @@ In addition, you will need to rename disk image 2 to disk1,
 disk image 3 to disk2, disk image 4 to disk3 and disk image 5 to disk4.
 
 
-3.17) Riven game notes:
------ -------------------------------
+3.17) Quest for Glory notes:
+----- ----------------------
+It is possible to import characters, beginning with Quest for Glory II,
+from past games to future games and continue from the stats earned
+from those games.
+
+For example, a character can be imported from Quest for Glory I directly
+to Quest for Glory III without having to necessarily play
+Quest for Glory II.
+
+Characters cannot be imported from future games to past games, nor can
+a character be imported to the same game that was just completed.
+In other words, a character from Quest for Glory II cannot be imported
+into Quest for Glory II.
+
+If you want to use a saved character from the original sierra interpreter,
```

Comment:
s/sierra/Sierra/ ?

Reply:
Yes, there is always one thing i miss :-)